### PR TITLE
Refactoring init stop point

### PR DESCRIPTION
**Pull request by datanel**
Refactoring init stop point 

Instead of calling 3 times the same stop point request, we can do it once

### CODE REVIEW

**Comment by datanel** [comment]
@kun2985 @dvdn anyone can check this PR please.
I think this PR deprecates https://github.com/CanalTP/MttBundle/pull/73

**Comment by vincentlepot** [comment]
jenkins test this please

File: Services/StopPointManager.php [hunk 1]
```diff
@@ -33,34 +30,28 @@ public function findByExternalId($externalId)
...
-    private function initStopPointCode($externalCoverageId)
-    {
-        $codes = $this->navitia->getStopPointCodes(
-            $externalCoverageId,
-            $this->stopPoint->getExternalId()
-        );
-        $this->stopPoint->setCodes($codes);
+        $administrativeRegion = isset($navitiaStopPoint->stop_points[0]->administrative_regions)
+            ? $navitiaStopPoint->stop_points[0]->administrative_regions[0]->name
+            : null;
```
Comment (dvdn) [inline Services/StopPointManager.php]: Is 'null' ok for default ?

**Comment by dvdn** [comment]
All tests pass ok (by rebasing on 'fix-tests' ;))

**Comment by datanel** [comment]
Jenkins retest this please

**Comment by vincentlepot** [comment]
Jenkins retest this please

**Pull request merged**